openedx/features/enterprise_support/tests/test_signals.py

```diff
@@ -0,0 +1,70 @@
+"""Tests of email marketing signal handlers."""
+import logging
+import ddt
+from django.contrib.sites.models import Site
+from django.test import TestCase
+from django.test.client import RequestFactory
+from mock import patch
+
+from email_marketing.models import EmailMarketingConfiguration  # pylint: disable=import-error
+from student.tests.factories import UserFactory
+from openedx.features.enterprise_support.tests.factories import EnterpriseCustomerFactory, EnterpriseCustomerUserFactory
+from openedx.features.enterprise_support.signals import update_email_marketing_user_with_enterprise_flag
+
+log = logging.getLogger(__name__)
+
+LOGGER_NAME = "enterprise_support.signals"
+
+TEST_EMAIL = "test@edx.org"
+
+
+def update_email_marketing_config(enabled=True, key='badkey', secret='badsecret', new_user_list='new list',
+                                  template='Welcome', enroll_cost=100, lms_url_override='http://testserver'):
+    """
+    Enable / Disable Sailthru integration
+    """
+    return EmailMarketingConfiguration.objects.create(
+        enabled=enabled,
+        sailthru_key=key,
+        sailthru_secret=secret,
+        sailthru_new_user_list=new_user_list,
+        sailthru_welcome_template=template,
+        sailthru_enroll_template='enroll_template',
+        sailthru_lms_url_override=lms_url_override,
+        sailthru_get_tags_from_sailthru=False,
+        sailthru_enroll_cost=enroll_cost,
+        sailthru_max_retries=0,
+        welcome_email_send_delay=600
+    )
```

Comment:
Do you need to set up an email marketing config in order to run the test suggested below?